data/SampleData/Json/bundle.json

```diff
@@ -0,0 +1,302 @@
+{
+  "resourceType": "Bundle",
```

Comment:
The test data directory and file names aren't all that clear on the intended test case.  Should we split by input / output types? From FHIR or To FHIR?